tests/cases/fourslash/codeFixPromiseToAsyncCatch_noBrackets.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference path='fourslash.ts' />
+
+// @target: es6
+
+////function [|f|]():Promise<void> {
+////    return fetch('http://yahoo.com').then(result => console.log(result)).catch(err => console.log(err));
+////}
+
+verify.getSuggestionDiagnostics([{
+    message: "This may be converted to use async and await.",
+    code: 80006,
+}]);
+
+
+verify.codeFix({
+    description: "Convert to use async and await",
+    index: 0,
+    newFileContent:
+`async function f():Promise<void> {
+    try {
+        let result = await fetch('http://yahoo.com');
+        console.log(result);

```

Comment:
Since the original source is a concise arrow (i.e. `result => console.log(result)`, rather than `result => { console.log(result); }`, its important to be sure we await the result of calling `console.log()` (since this could be any arbitrary expression).

As a result, this line should be: `await console.log(result);`, unless you are eliding the `await` because the return type is `void`. If that is the case, then I'd like to see a test for an arrow whose result is non-`void`.